docs/application/native/guides/text-input/autofill-manager.md

```diff
@@ -166,7 +148,27 @@ void set_autofill_service(autofill_manager_h amh, const char *autofill_service_a
 }
 ```
 
+## Release Resources
+
+After you have finished working with the Autofill Manager library, destroy the the Autofill Manager handle:
+
+   ```c
+   void
+   destroy_autofill_manager_handle(autofill_manager_h amh)
+   {
+       int ret;
+
+       ret = autofill_manager_destroy(amh);
+       if (ret != AUTOFILL_ERROR_NONE)
+           /* Error handling */
+   }
+   ```
+
+   > **Note**
+   >
+   > You must not use the `autofill_manager_destroy()` function in a callback function.
+
 ## Related Information
 
-* Dependencies
-  - Tizen 5.5 and Higher
+- Dependencies
```

Comment:
Question: 
Please can you let me know whether the dependencies support for both wearable and mobile?

Reply:
yes, it support mobile, wearable, TV since Tizen 5.5.
Do I have to mention like "Tizen 5.5 and Higher for Mobile and Wearable"?

Reply:
@imfine98, 
In case of Native guides we must not write it in a single line:
Please change as:
- Tizen 5.5 and Higher for Mobile 
- Tizen 5.5 and Higher for Wearable
- Tizen 5.5 and Higher for TV

Reply:
TV profile does not provide CAPI, so I write like below as you suggested:
Tizen 5.5 and Higher for Mobile
Tizen 5.5 and Higher for Wearable

Reply:
@imfine98, okay sure

Reply:
Fixed.